src/scene_battle_rpg2k3.cpp

```diff
@@ -60,10 +60,13 @@ Scene_Battle_Rpg2k3::Scene_Battle_Rpg2k3(const BattleArgs& args) :
 
 void Scene_Battle_Rpg2k3::Start() {
 	Scene_Battle::Start();
+	Game_Interpreter_Battle::InitBattle();
 	InitBattleCondition(Game_Battle::GetBattleCondition());
 	CreateEnemySprites();
 	CreateActorSprites();
 
+	Game_Battle::GetInterpreterBattle();
```

Comment:
This function has no side effects and the return value is ignored. Can be removed?